testonly/hammer/hammer.go

```diff
@@ -220,8 +220,9 @@ func HitMap(ctx context.Context, cfg MapConfig) error {
 	wg.Add(1)
 	go func() {
 		defer wg.Done()
-		glog.Infof("%d: start main goroutine", s.cfg.MapID)
-		count, err := s.performOperations(ctx, done)
+		w := newWorker(s.cfg, rand.New(s.cfg.RandSource))
+		glog.Infof("%d: start main goroutine", cfg.MapID)
```

Comment:
Is `s.cfg` the same thing as `cfg`? Use one or the other consistently?

Reply:
I thought I'd already fixed all these occurrences. I did a lot of back and forth at around this point of the refactoring so I probably undid it. Good spot.